Updating info alert to be more noticeable

kvisca opened the pull request:
As per discussion with @dvnkshl, the info alerts are not noticeable when set to inline because of the grey coloring. Updating it to our blue to make it more noticeable. 

**Preview:**
From
![Screen Shot 2021-04-05 at 4 08 59 PM](https://user-images.githubusercontent.com/36721153/113621116-41669700-9629-11eb-8120-5873a94c1f2d.png)
To
![Screen Shot 2021-04-05 at 4 08 36 PM](https://user-images.githubusercontent.com/36721153/113621162-4f1c1c80-9629-11eb-9697-8a2e612293fa.png)

**Example Integration:**

From
![Screen Shot 2021-04-05 at 4 10 05 PM](https://user-images.githubusercontent.com/36721153/113621427-9acec600-9629-11eb-919f-2f70e60296de.png)

To
![Screen Shot 2021-04-05 at 4 11 14 PM](https://user-images.githubusercontent.com/36721153/113621419-97d3d580-9629-11eb-9313-ecdc6ecff4a0.png)

robbieaverill reviewed (approved):
Can you update Figma too please?

kvisca commented:
> Can you update Figma too please?

Good call.

kvisca commented:
Fixed a few issues with borders being off on the sticky variant and added in the grey for both inline and sticky as a new variant called `message`.

**Sticky:**
![Screen Shot 2021-04-05 at 5 55 43 PM](https://user-images.githubusercontent.com/36721153/113631958-467f1280-9638-11eb-96ca-ae5892fcab28.png)

**Inline:**
![Screen Shot 2021-04-05 at 5 55 39 PM](https://user-images.githubusercontent.com/36721153/113631968-4a129980-9638-11eb-87df-87a265eb58ef.png)

robbieaverill commented on src/components/ChecAlert.vue:
```diff
@@ -111,6 +112,10 @@ $alert-inline-colors: (
   ),
   'info': (
+    'background': 'blue',
+    'border': 'blue-500',
```
The other colour variants (excluding gray) all use 400, do you want to keep 500 here?

ScopeyNZ replied:
Nice spotting. 🙈

kvisca replied:
Yea it makes it stand out a bit more and matches with the other variants a bit better.
